lib/components/table.vue

```diff
@@ -75,7 +75,8 @@
 
 <script>
     import { warn } from '../utils';
-    import { keys } from '../utils/object.js'
+    import { keys } from '../utils/object.js';
+    import { listenOnRootMixin } from '../mixins'
```

Comment:
Goodness, look at me. I tried to fight my editor prefs and add a semicolon, and then I forget it in the very next line!